setup.cfg

```diff
@@ -1,10 +1,66 @@
 [metadata]
-package_name = jwst
-description = JWST Pipeline
-author = STScI
-license = BSD
-edit_on_github = False
-github_project = spacetelescope/jwst
+name = jwst
+description = Python library for science observations from the James Webb Space Telescope
+long_description = file: README.md
```

Comment:
This is what the front page on PyPI will be. Do we want to put there all that is in the `README`?
All the information about DMS builds and links to Jenkins?

Reply:
That's a good question.  Maybe we should break out our README.md into chunks and only include one of those chunks on the PyPi page?

Reply:
@hbushouse, @stscieisenhamer, any opinions on what sections in our current README, if anything, should we not have on the PyPi landing page?

Some examples of some other README files for packages we all use:

https://pypi.org/project/pytest/
https://pypi.org/project/numpy/
https://pypi.org/project/astropy/
https://pypi.org/project/matplotlib/
https://pypi.org/project/asdf/
https://pypi.org/project/crds/

As you can see, they run the gamut of just some links, to the full help of how to use the package.

Reply:
Definitely leave out the section with the DMS builds table and the regression tests. What about unit tests - I guess that's still of use to the public? What about JupyterHub - is that info for the public?

Reply:
In order to break up the README into chunks, we would have to switch from markdown to restructured text.  Perhaps just a link to our existing `README.md` would be OK for the long description on PyPi?

Reply:
So for this PR, I'm just removing the long description and adding in the links to our github page and documentation on RTD in the standard PyPi way.  We can rethink how we want to present this information on PyPi, if needed, in a separate PR and maybe after we've already at least done a release to PyPi.  I think this will suffice for now, while not putting on PyPi all the gory detail we have in our current README.md.